contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/Invoice.java

```diff
@@ -92,7 +92,7 @@ public class Invoice {
     private Customer counterparty;
 
     @OneToMany
-    @JoinColumn(name = "__invoice_id__orders", foreignKey = @ForeignKey(foreignKeyDefinition = "foreign key (\"invoice\") references \"invoice\" ON DELETE set NULL"))
+    @JoinColumn(name = "__invoice_id__orders", foreignKey = @ForeignKey(foreignKeyDefinition = "foreign key (\"__invoice_id__orders\") references \"invoice\" ON DELETE set NULL"))
```

Comment:
Should this be part of this PR?

Reply:
This is also a fix, before, the foreign key was not set up correctly (and hibernate ignored the incorrect definition, just logging a warning to the console all the time)